app/models/user.rb

```diff
@@ -7,58 +7,32 @@ class User < ApplicationRecord
 
   # NOTE: @citizen428 This is temporary code during profile migration and will
   # be removed.
-  # rubocop:disable Metrics/BlockLength
   concerning :ProfileMigration do
     included do
-      PROFIE_FIELDS =
-        (Profiles::ExtractData::DIRECT_ATTRIBUTES + Profiles::ExtractData::MAPPED_ATTRIBUTES.values).freeze
-
       # NOTE: There are rare cases were we want to skip this callback, primarily
       # in tests. `skip_callback` modifies global state, which is not thread-safe
       # and can cause hard to track down bugs. We use an instance-level attribute
       # instead. See `spec/factories/profiles.rb` for an example.
       attr_accessor :_skip_creating_profile
 
+      # NOTE: used for not sync-ing back data to profiles when the profile got
+      # updated first. This will eventually be removed:
+      attr_accessor :_skip_profile_sync
```

Comment:
This allows us to skip the sync from user to profile when updating doing it through the service object (in which case the sync direction is profile -> user). 

I was debating removing this sync direction entirely, but 

1. I wanted to be able to deploy this before we change the `/settings/profile` page to use the service object and
2. ensure that there are no other places where this gets set, so we don't risk getting out of sync.